lib/fog/compute/ecloud.rb

```diff
@@ -0,0 +1,825 @@
+                          internet_service = {
+                            :id           => internet_service_id,
+                            :href         => "/cloudapi/ecloud/internetservices/#{internet_service_id}",
+                            :name         => Fog::Mock.random_letters(6),
+                            :type         => "application/vnd.tmrk.cloud.internetService",
+                            :public_ip_id => public_ip_id,
+                            :Links => {
+                              :Link => [
+                                Fog::Ecloud.keep(public_ip, :href, :name, :type),
+                              ],
+                            },
+                            :NodeServices => {
+                              :NodeService => [
+                              ]
+                            },
+                          }
+
+                          node_service = {
+                            :id                  => node_service_id,
+                            :href                => "/cloudapi/ecloud/nodeservices/#{node_service_id}",
+                            :name                => Fog::Mock.random_letters(6),
+                            :type                => "application/vnd.tmrk.cloud.nodeService",
+                            :internet_service_id => internet_service_id,
+                            :Links               => {
+                              :Link => [
+                                Fog::Ecloud.keep(internet_service, :href, :name, :type)
+                              ],
+                            },
+                          }
+
+                          internet_service[:NodeServices][:NodeService].push(node_service)
+                          public_ip[:InternetServices][:InternetService].push(internet_service)
+
+                          network = {
+                            :id               => network_id,
+                            :href             => "/cloudapi/ecloud/networks/#{network_id}",
+                            :name             => "#{network_ip}/#{Fog::Mock.random_numbers(2)}",
+                            :type             => "application/vnd.tmrk.cloud.network",
+                            :Address          => network_ip,
+                            :NetworkType      => "Dmz",
+                            :BroadcastAddress => network_ip,
+                            :GatewayAddress   => network_ip,
+                            :environment_id   => environment_id,
+                            :Links => {
+                              :Link => [
+                                Fog::Ecloud.keep(environment, :href, :name, :type),
+                              ]
+                            },
+                            :IpAddresses => {
+                              :IpAddress => [],
+                            },
+                          }
+
+                          ip_address = {
+                              :id         => ip_address_id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.